src/Language/Fortran/Util/ModFile.hs

```diff
@@ -217,6 +217,9 @@ decodeModFiles' = fmap (map snd) . decodeModFiles
 combinedModuleMap :: ModFiles -> FAR.ModuleMap
 combinedModuleMap = M.unions . map mfModuleMap
 
+localisedModuleMap :: FAR.ModuleMap -> FAR.ModuleMap
+localisedModuleMap = M.map (M.filter (not . FA.isImported . snd))
```

Comment:
this threw me for a bit thanks to the deep type synonyms (`ModuleMap` storing a `ModEnv` which itself is actually a `Map`) -- could perhaps clarify what it's doing in a comment

Reply:
that is, I saw `map . M.filter` and my first thought was "isn't this just a filter operation", but we're actually filtering the maps inside our top-level map

Reply:
Added a comment!

Reply:
Thanks!